src/msg/simple/Pipe.cc

```diff
@@ -332,7 +334,7 @@ int Pipe::accept()
     goto fail_unlocked;
   }
   {
-    bufferptr tp(sizeof(peer_addr));
+    bufferptr tp(sizeof(ceph_entity_addr));
```

Comment:
I think the real issue is that the on-wire protocol is defined by 
struct ceph_entity_addr; the fact that entity_addr_t was the same size was 
incidental.  The code already goes through encode/decode, so I think with 
this change it's most correct?